Resources/Textures/_Impstation/Structures/cryostorage.rsi/meta.json

```diff
@@ -0,0 +1,27 @@
+{
+    "version": 1,
+    "license": "CC-BY-SA-3.0",
+    "copyright": "Taken from SS220 at commit https://github.com/SerbiaStrong-220/space-station-14/commit/a55288dc926162c28665b542304a6ac1f8e1b91e, then modified by AftrLite (Github).",
```

Comment:
SS220 is okay with us editing their art assets, right? i know they are really picky on sharing code